pyproject.toml

```diff
@@ -79,5 +79,8 @@ aact = { git = "https://github.com/ProKil/aact" , branch = "main" }
 testpaths = ["tests"]
 python_files = "test_*.py"
 
+[tool.hatch.metadata]
+allow-direct-references = true
```

Comment:
what's this for?

Reply:
Oh this allows me to declare a dependency directly from GitHub